app/Observers/TripObserver.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+namespace App\Observers;
+
+use App\Enum\Report\ReportableSubject;
+use App\Enum\Report\ReportReason;
+use App\Models\Trip;
+use App\Services\ReportService;
+use Illuminate\Support\Facades\Log;
+
+class TripObserver
+{
+    public function created(Trip $trip): void {
+        // check if trip is out of allowed types and create an admin report if so
+        $triggerWords = ['auto', 'fuss', 'fuß', 'fahrrad', 'foot', 'car', 'bike'];
```

Comment:
Bitte in nen eigenen Service auslagern. Das ist von der Funktionalität schon fast generisch genug, dass wir es an mehreren Stellen gut nutzen könnten